src/Win32Tests/Win32ApisTests.cs

```diff
@@ -1,267 +1,223 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-using System.Collections.Generic;
+using Axe.Windows.SystemAbstractions;
 using Axe.Windows.Win32;
-using Microsoft.QualityTools.Testing.Fakes;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Microsoft.Win32.Fakes;
+using Moq;
+using System;
+using System.Collections.Generic;
 
 namespace Axe.Windows.Win32Tests
 {
     [TestClass]
     public class Win32ApisUnitTests
     {
+        private readonly Mock<IMicrosoftWin32Registry> _registryMock;
+        private readonly Win32Helper _win32Helper;
+
+        public Win32ApisUnitTests()
+        {
+            _registryMock = new Mock<IMicrosoftWin32Registry>(MockBehavior.Strict);
+                _win32Helper = new Win32Helper(_registryMock.Object);
+        }
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            _registryMock.Reset();
```

Comment:
>_registryMock [](start = 12, length = 13)

This pattern requires test to run sequentially. That's probably OK unless someone uses ReSharper's test engine (which has the option to run them in parallel)

Reply:
Noted. Given the existence of the TestInitializeAttribute and it's serial implications though, I feel comfortable with this.